packages/botonic-core/src/models/events/message/media.ts

```diff
@@ -0,0 +1,32 @@
+import { BotonicMessageEvent, MessageEventType } from '.'
+import { WithButtons } from './buttons'
+
+export interface AudioMessageEvent
+  extends BotonicMessageEvent<MessageEventType.audio>,
+    WithButtons {
+  src: string
+}
+
+export interface ImageMessageEvent
+  extends BotonicMessageEvent<MessageEventType.image>,
+    WithButtons {
+  src: string
+}
+
+export interface DocumentMessageEvent
+  extends BotonicMessageEvent<MessageEventType.document>,
+    WithButtons {
+  src: string
+}
+
+export interface VideoMessageEvent
+  extends BotonicMessageEvent<MessageEventType.video>,
+    WithButtons {
+  src: string
+}
+
+export type MediaMessageEvent =
+  | AudioMessageEvent
+  | ImageMessageEvent
+  | DocumentMessageEvent
+  | VideoMessageEvent
```

Comment:
Since all media events have the `src` attribute, would it make sense to define `MediaMessageEvent` as:
```
export interface MediaMessageEvent extends WithButtons {
  src: string
}
```
And then declare the specific media events like this?:
```
export interface VideoMessageEvent extends MediaMessageEvent, BotonicMessageEvent<MessageEventType.video> {}
```

Reply:
applied differently! let me know when you take it a look 👍